.travis.yml

```diff
@@ -133,6 +133,58 @@ matrix:
       after_failure:
         - aws s3 cp . s3://mapbox/mapbox-gl-native/render-tests/$TRAVIS_JOB_NUMBER --recursive --exclude "*" --include "*.trace"
 
+    # EGL - Node v6 - Clang 3.9 - Debug
```

Comment:
Unless there's a compelling reason to test the Node v6 debug build independently from Node v4 debug and Node v6 release, we can probably omit this row from the matrix.